cmd/config.go

```diff
@@ -323,23 +359,25 @@ type LogDescription struct {
 
 // GRPCClientConfig contains the information needed to talk to the gRPC service
 type GRPCClientConfig struct {
-	ServerAddresses       []string
+	ServerAddresses []string
+	Timeout         ConfigDuration
+	// Deprecated. Use TLSConfig instead. TODO(#2472): Delete these.
 	ServerIssuerPath      string
 	ClientCertificatePath string
 	ClientKeyPath         string
-	Timeout               ConfigDuration
 }
 
 // GRPCServerConfig contains the information needed to run a gRPC service
 type GRPCServerConfig struct {
-	Address               string `json:"address" yaml:"address"`
-	ServerCertificatePath string `json:"serverCertificatePath" yaml:"server-certificate-path"`
-	ServerKeyPath         string `json:"serverKeyPath" yaml:"server-key-path"`
-	ClientIssuerPath      string `json:"clientIssuerPath" yaml:"client-issuer-path"`
+	Address string `json:"address" yaml:"address"`
```

Comment:
These YAML fields are not used anywhere after we removed the standalone CAA checker, they don't necessarily  need to be removed now though.